### ISSUE TITLE
Errors when deleting event

### ISSUE THREAD

**Issue by fr4ncis3** [insanum/gcalcli#321]
Hi all.
Working with gcalcli from a couple of days. Nice project! Thanks for your work!

Using ver:
gcalcli v4.0.0a3 (Eric Davis, Brian Hartvigsen) from debian testing on a debian stable.
It works pretty well.

Only have problems when I try to delete events. I get the following errors:

#---------------
francis3@vps437939:~$ gcalcli search 'EVENT'

2017-10-25           TEST-EVENT

francis3@vps437939:~$ gcalcli delete 'EVENT'

2017-10-25           TEST-EVENT
Delete? [N]o [y]es [q]uit: Traceback (most recent call last):
  File "/usr/bin/gcalcli", line 2500, in <module>
    BowChickaWowWow()
  File "/usr/bin/gcalcli", line 2472, in BowChickaWowWow
    FLAGS.iamaexpert, eStart, eEnd)
  File "/usr/bin/gcalcli", line 1838, in DeleteEvents
    yearDate=True, work=self._DeleteEvent)
  File "/usr/bin/gcalcli", line 1530, in _IterateEvents
    work(event)
  File "/usr/bin/gcalcli", line 1348, in _DeleteEvent
    val = raw_input()
NameError: name 'raw_input' is not defined
#-------------

Any idea?

Thx

**Comment by fr4ncis3** [comment]
Using ver. 
gcalcli v3.4.0 (Eric Davis, Brian Hartvigsen)
from debian stable and needed python packages from stable solved the issue.

**Comment by tresni** [comment]
3.4 is old, 4.0.0 is where development (when I do it :P ) is happening.  My guess is you are using python3 and I missed that in doing my tests.

**Comment by fr4ncis3** [comment]
I've ran some test

On a plain Debian buster / testing

----------
gcalcli v4.0.0a3 (Eric Davis, Brian Hartvigsen)
Python 2.7.14

I get the error

--------

On a plain Debian stretch / stable

gcalcli v3.4.0 (Eric Davis, Brian Hartvigsen)
Python 2.7.13

Gcalcli works fine in deleting events
--------

Thanks for your answer

**Comment by fr4ncis3** [comment]
On a debian testing / buster and

gcalcli v4.0.0a3 (Eric Davis, Brian Hartvigsen)

I get the error deleting the events with the following python versions

Python 2.7.14

Python 3.5.4

Python 3.6.3

**Comment by photex** [comment]
I get this same error when trying to edit events. 
```
➜  ~ gcalcli --version
gcalcli v4.0.0a3 (Eric Davis, Brian Hartvigsen)
➜  ~ more /etc/issue
Ubuntu 17.10 \n \l
```

**Comment by jcrowgey** [comment]
I've been working on the alpha branch (which is now in master) for a couple of months.  I *belive* that I've got things set up to properly support python 27 and python 35.  I don't think there should be any issues  using python36 either, although I haven't tested against it.

Please reopen this if you guys can still repro.  I'm not able to repro this on the current master using py3 or py2 (using debian).

**Comment by nadir-yttrium** [comment]
I still have the issue.

```
Edit?
[N]o [s]ave [q]uit [t]itle [l]ocation [w]hen len[g]th [r]eminder [d]escr: Traceback (most recent call last):
  File "/usr/bin/gcalcli", line 2538, in <module>
    BowChickaWowWow()
  File "/usr/bin/gcalcli", line 2520, in BowChickaWowWow
    gcal.EditEvents(_u(FLAGS.text))
  File "/usr/bin/gcalcli", line 1887, in EditEvents
    yearDate=True, work=self._EditEvent)
  File "/usr/bin/gcalcli", line 1568, in _IterateEvents
    work(event)
  File "/usr/bin/gcalcli", line 1416, in _EditEvent
    val = raw_input()
NameError: name 'raw_input' is not defined
```

Antergos Linux
gcalcli v4.0.0a4
Python 3.7.2

**Comment by jcrowgey** [comment]
This issue is indeed fixed, but after v4.0.0a4 (it was fixed in a commit be03171, see #400).

Don't worry, there will be a new release soon.  In the meantime, feel free to install from this repo.
